manifests/d/Devolutions/Workspace/2024.1.1.0/Devolutions.Workspace.installer.yaml

```
@@ -0,0 +1,30 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Devolutions.Workspace
+PackageVersion: 2024.1.1.0
+InstallerLocale: en-US
+Platform:
+- Windows.Desktop
+InstallerType: msi
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Protocols:
+- credential
+- http
+- https
+ProductCode: '{A4BCFDF0-CBDC-4622-83C4-EB17E6C98CE2}'
+ReleaseDate: 2024-03-13
+AppsAndFeaturesEntries:
+- UpgradeCode: '{EA58A259-5C39-43E8-9183-EAAB09293D3F}'
+- DisplayVersion: 24.1.1
```

```suggestion
- DisplayVersion: 24.1.1.0
```
Sorry, but I believe the DisplayVersion has to be an exact match, including the trailing zero.